src/routes/+page.svelte

```diff
@@ -35,7 +36,8 @@
     
 </section>
 
-<Searsh />
+<Search 
+cardData={data.uitleengeschiedenis1}/>
```

Comment:
Het werkte eerst niet omdat je nu 2 querys doet in de page.server.js. Nu moet je dit schrijven om de search werkend te krijgen.

`cardData={data.hygraphData.uitleengeschiedenis1}/>`